blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -418,17 +420,10 @@ public void testCompressedRequests() throws Exception{
     @Test
     public void testMultiTenantBatching() throws Exception{
 
-        long start = System.currentTimeMillis() - 120000;
-        long end = System.currentTimeMillis() + 120000;
+        long start = System.currentTimeMillis() - TIME_DIFF;
+        long end = System.currentTimeMillis() + TIME_DIFF;
 
-        URIBuilder builder = getMetricsURIBuilder()
-                .setPath("/v2.0/acTest/ingest/multi");
-        HttpPost post = new HttpPost(builder.build());
-        String content = JSONMetricsContainerTest.generateMultitenantJSONMetricsData();
-        HttpEntity entity = new StringEntity(content,
-                ContentType.APPLICATION_JSON);
-        post.setEntity(entity);
-        HttpResponse response = client.execute(post);
+        HttpResponse response = httpPost( TID, postMultiPath, JSONMetricsContainerTest.generateMultitenantJSONMetricsData() );
```

Comment:
When we post multiple metrics to ingest/multi, some with valid timestamp and some with delayed/future timestamp which are supposed to fail, what happens to the overall request?